services/user.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    try:
+        return get_user_model().objects.get(pk=user_id)
+    except ObjectDoesNotExist:
+        return None
```

Comment:
Returning `None` from `get_user` when a user is not found can lead to `AttributeError` in the `update_user` function. Consider raising an exception or returning a more informative error message.